Dockerfile

```diff
@@ -1,11 +1,23 @@
-FROM lsiobase/alpine:3.12
+FROM lsiobase/alpine:3.11 as downloader
```

Comment:
Worth making this the same Alpine version, to save the duplicate container downloading.

```suggestion
FROM lsiobase/alpine:3.12 as downloader
```